packages/grid/_modules_/grid/components/containers/GridToolbarContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { classnames } from '../../utils';
+
+type GridToolbarProps = React.HTMLAttributes<HTMLDivElement>;
+
+export const GridToolbarContainer = React.forwardRef<HTMLDivElement, GridToolbarProps>(
```

Comment:
A bit confusing whether this is the Grid's `Toolbar` or `ToolbarContainer`. Both names are used in the file.

Reply:
Ah yes I forgot to rename the props